src/api/endpoints/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .category import router as category_router  # noqa
```

Comment:
Мы делаем импорты иначе:
```suggestion
from .category import router as category_router

__all__ = ("category_router",)
```
Вообще я думал о такой структуре:
```
api/
|___routers/
     |__base_router.py # Тут был бы базовый роутер - как сейчас в файле api/router.py
     |__some_router.py
```
**Но это лишнее**, у нас всего будет два хендлера за весь проект, легче обойтись одним роутером и файлом.